docs/contributing/marketplace.md

```diff
@@ -48,8 +50,29 @@ This article walks you through the process of contributing content from the Cort
 8. You can now modify the files changed in the pull request as part of the review process.
 
 
+## Resubmit a Content Pack
+
```

Comment:
looks great !
perhaps its worth explaining what resubmitting is (e.g. updating an open PR...)?